### ISSUE TITLE
admin cosmetic change

### ISSUE THREAD

**Issue by bhbraswell** [Applied-GeoSolutions/geokit#109]
So this might sound crazy but  I think there is a way to make the admin interface look much nicer by doing something about the combination of the purply and the gray colors, which slam into one another in a painful way in the upper left corner.

<img width="571" alt="yuck" src="https://cloud.githubusercontent.com/assets/3209308/26420091/a337108e-408f-11e7-977c-97eeaa2f71c5.png">

I know I'm not an aesthetician, and I can't even match my socks, but I think this will help: either change the purply color to make it more gray, or change the gray color to make it more purply.

That is all.

**Comment by demiurg** [comment]
@bglendenning stop slamming purple and gray colors and learn to aesthetic

**Comment by bglendenning** [comment]
This is actually one of my favorite color schemes I've done, and I disagree that they clash, but it won't hurt my feelings to play with the colors until we find something you like.

I'll provide some images with color tweaks that account for your comments.

**Comment by bhbraswell** [comment]
Sorry, nevermind. If you think its good and I think it's bad, then it's probably good.

**Comment by demiurg** [comment]
I think it's good, but but an acquired taste. I think the main problem overall is that theme is not contrasting enough. It might be aesthetic in some radical sense, most of all it should have mass appeal and approachability, as opposed to strong reactions, one way, or another.

**Comment by demiurg** [comment]
If there are resources to further explore this, I don't see any harm in trying to come closer to perfection?

**Comment by bhbraswell** [comment]
Sure, up to you guys. I think that since previously logged in admin users will always go directly to the admin, then it should be treated as the main UI rather than a "behind the scenes" kind of interface.

**Comment by demiurg** [comment]
I agree with that. I also think it's a bit plain too, while we're talking about first exposure. Perhaps we could add some user guide type stuff of 'Welcome' graphics or hero header.

**Comment by demiurg** [comment]
Also, I realized shifting some welcoming photos to more purplish shade would be cool. Also, everyone loves purple.

**Comment by bglendenning** [comment]
I'm not sure about the guide type stuff. Maybe it's better for another issue if there's going to be some brain storming. I like the idea of showing a check-list of items completed so far, with some kind of status indicator that shows whether or not visualizations can be created, and, if not, what steps are required to get to that point. We also talked about the potential for an admin landing page. Maybe now that this milestone has been reached, looking into the landing page is feasible.

If there's a particular image you would like to see in the landing page carousel, send it to me and we can try it out.

Back to this actual issue: I'll provide some variations of colors. It shouldn't be a time-intensive process.

**Comment by bglendenning** [comment]
I'm providing 3 slightly modified versions of the theme. All three shift from the slightly grey/green header to a more purple header.

![purpler](https://cloud.githubusercontent.com/assets/8276428/26570845/3a722172-44d8-11e7-908b-cbae096c6327.png)

Above is the theme with a more purple header.

![purpler lighter](https://cloud.githubusercontent.com/assets/8276428/26570844/3a6fbc0c-44d8-11e7-9d76-a2ebb288e1c1.png)

Above is the same theme, but slightly lighter header.

![lighter](https://cloud.githubusercontent.com/assets/8276428/26570843/3a6f4952-44d8-11e7-9ba6-3afb0418a3ca.png)

Finally, the same theme, but with a slightly desaturated & lightened sidebar.

**Comment by demiurg** [comment]
I can't tell any difference, are you sure it's not same file?

**Comment by bglendenning** [comment]
The differences are subtle. In the current theme, the header "grey" color is tinted slightly green. In these latest, they're tinted more toward the red/purple. It's easier to spot the differences if you open the images up in new tabs, and cycle through them.

Or are you being funny? I can never tell.

**Comment by demiurg** [comment]
I couldn't tell, maybe my screen brightness was too low. I think that also means it's too subtle? I can see that I do prefer the top one now.
